resources/views/device-group/index.blade.php

```diff
@@ -57,6 +57,39 @@ class="fa fa-trash" aria-hidden="true"></i></button>
             </div>
         </div>
     </div>
+    <div class="container-fluid">
+        <div id="manage-device-groups-panel" class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">
+                @lang('Ungrouped Devices') ({{ $ungrouped_devices->count() }})
+                </h4>
+            </div>
+            <div class="panel-body">
+                <div class="table-responsive">
+                    <table id="ungrouped-devices-table" class="table table-condensed table-hover">
+                        <thead>
+                        <tr>
+                            <th>@lang('Vendor')</th>
+                            <th>@lang('Device')</th>
+                            <th>@lang('Platform')</th>
+                            <th>@lang('Operating System')</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        @foreach($ungrouped_devices as $device)
+                            <tr id="row_{{ $device->device_id }}">
+                                <td><img alt="{{ $device->os }} "src="{{ asset($device->icon) }}" width="32px" height="32px" title="{{ $device->os }}"></td>
+                                <td><a class="list-device" href="{{ url("device/device=$device->device_id") }}"><b>{{ $device->hostname }}</b></a><br>{{$device->sysName}}</td>
+                                <td>{{ $device->hardware }}</td>
+                                <td>{{ $device->os }} {{ $device->version }} ({{ $device->features }})</td>
```

Comment:
Probably don't show () if features is empty...

Reply:
changed